modules/python/pylib/setup.py

```diff
@@ -21,7 +21,7 @@
 #
 #############################################################################
 
-from distutils.core import setup
+from setuptools import setup
```

Comment:
Note: We could list dependencies here under `install_requires=[]`.

Reply:
I know and the current solution does that. It's somewhat ugly though (as pip insists on installing my package and its dependencies in the same directory), but look at the current solution and let me know how you like it. Let's discuss if a discussion is needed.